### PR TITLE
SplashReagents now checks if the bottle has reagents

### PR DESCRIPTION
**Pull request by Mervill**
`reagents` can be null in some cases, so we check if it exists before splashing reagents

### CODE REVIEW

**Comment by optimumtact** [comment]
See I wish we knew why reagents could be null at this point

**Comment by Mervill** [comment]
/datum/reagents/Destroy()

**Comment by Mervill** [comment]
This PR combined with [22855] should fix any runtimes generated by those kind of vending-machine themed explosions

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 1]
```diff
@@ -124,7 +124,7 @@
 	return
 
 /obj/item/weapon/reagent_containers/food/drinks/bottle/proc/SplashReagents(var/mob/M)
-	if(src.reagents.total_volume)
+	if(src.reagents && src.reagents.total_volume)
```
Comment (lzimann) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm]: src. reee etc

**Comment by Cyberboss** [comment]
@Mervill This function is actually a dupe of the splash function in `reagent_containers` I'm refactoring bottles to use it in my PR

**Comment by Mervill** [comment]
okay

**Pull request closed without merging**